en_US/rule/backend_oracle.md

```diff
@@ -42,6 +42,8 @@ Fill in the resource configuration:
 
 ![image](./assets/rule-engine/oracle_action_3.png)
```

Comment:
Unfortunately, the Chinese fonts do not display correctly in my machine to allow me to take a new screenshot of the resource creation for the Chinese translation. :crying_cat_face: